### PR TITLE
Changes to make adding to home screen better

### PR DESCRIPTION
**Pull request by Veldrovive**
I added icons that will appear on the home screen (Not final, I'm no graphic designer so by all means change it). I also added a script that prompts the user to add the site to their home screen.

### CODE REVIEW

**Comment by Veldrovive** [comment]
Don't tell me to fix the codacy issues. It's someone else's code and I don't want to break it by changing stuff.

**Comment by HeroCC** [comment]
Icons are fine, but I don't like the huge js / CSS homescreen library. Instead, use something like this for a native integration: https://developers.google.com/web/fundamentals/engage-and-retain/app-install-banners/

File: index.html [hunk 1]
```diff
@@ -14,6 +14,17 @@
   <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="css/main.css">
 
+  <link rel="stylesheet" type="text/css" href="./css/addtohomescreen.css">
+  <script src="./js/addtohomescreen.min.js"></script>
+  <script>
+      addToHomescreen();
+  </script>
+
+  <link rel="apple-touch-icon" sizes="57x57" href="./img/ios-icon-57x57.png" />
```
Comment (HeroCC) [inline index.html]: These icons should have transparent backgrounds, not white. And, don't include the '.' in the path

File: index.html [hunk 2]
```diff
@@ -14,6 +14,17 @@
   <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="css/main.css">
 
+  <link rel="stylesheet" type="text/css" href="./css/addtohomescreen.css">
+  <script src="./js/addtohomescreen.min.js"></script>
```
Comment (HeroCC) [inline index.html]: We don't really need this, chrome supports it nativity and it adds a lot of bloat / code to maintain

File: index.html [hunk 3]
```diff
@@ -14,6 +14,17 @@
...
+  <script>
+      addToHomescreen();
```
Comment (HeroCC) [inline index.html]: Use as little script tags in HTML as possible, include this in main.js if we end up using it?

**Review by HeroCC — CHANGES_REQUESTED** [review]
Separate your PRs and make the icons transparent. Otherwise, thanks!

File: index.html (hunk 1 above)
Comment (HeroCC) [reply index.html]: And when previewing the SVG, it appears blank? Just a transparent image? Is that just me or did you upload the wrong thing?

Comment (Veldrovive) [reply index.html]: IOS has non transparent backgrounds so I made them non transparent. The SVG isn't blank, it seems like GitHub is simply not displaying it correctly.

**Comment by HeroCC** [comment]
I made a baseline for schedule push notifications, so the manifest stuff was superseded, can you make it just the icons?

**Pull request closed without merging**